CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 Recent changes to the [Chirp Arduino SDK](https://developers.chirp.io/docs).
 
+## v3.3.0 (beta)
+ - Added send-only support for cortex-m0plus (MKRZero, MKR Vidor 4000)
+ - Added support for cortex-m4 (Nano 33 Sense)
+ - Build v3.2.10
```

Comment:
Don't know if i wont update the build number to something like 3.3.0-rc1 as there are a few important changes + this is a kind of beta release

Reply:
Agreed

Reply:
I meant for the c-sdk build 🙈